packages/dds/map/package.json

```diff
@@ -96,6 +96,43 @@
   },
   "typeValidation": {
     "version": "2.0.0",
-    "broken": {}
+    "broken": {
+      "RemovedInterfaceDeclaration_IDirectoryClearOperation": {
+        "forwardCompat": false,
+        "backCompat": false
+      },
+      "RemovedInterfaceDeclaration_IDirectoryCreateSubDirectoryOperation": {
+        "forwardCompat": false,
+        "backCompat": false
+      },
+      "RemovedInterfaceDeclaration_IDirectoryDeleteOperation": {
+        "forwardCompat": false,
+        "backCompat": false
+      },
+      "RemovedInterfaceDeclaration_IDirectoryDeleteSubDirectoryOperation": {
+        "forwardCompat": false,
+        "backCompat": false
+      },
+      "RemovedTypeAliasDeclaration_IDirectoryKeyOperation": {
+        "forwardCompat": false,
+        "backCompat": false
+      },
+      "RemovedInterfaceDeclaration_IDirectorySetOperation": {
+        "forwardCompat": false,
+        "backCompat": false
+      },
+      "RemovedTypeAliasDeclaration_IDirectoryStorageOperation": {
+        "forwardCompat": false,
+        "backCompat": false
+      },
+      "RemovedTypeAliasDeclaration_IDirectorySubDirectoryOperation": {
+        "forwardCompat": false,
+        "backCompat": false
+      },
+      "RemovedInterfaceDeclaration_ILocalValue": {
+        "forwardCompat": false,
+        "backCompat": false
+      }
+    }
```

Comment:
Could it be related to this: https://github.com/microsoft/FluidFramework/pull/11011/files ? I'm not sure if this was released as 1.2

Reply:
seems to be related. it is part of 1.2 release

Reply:
@Josmithr was this a breaking change on main? Did you need to fix typetests?

Reply:
That PR strictly added items to the API surface that were already transitively included. It should have been strictly non-breaking.